kotsadm/pkg/apiserver/server.go

```diff
@@ -104,6 +104,7 @@ func Start() {
 	// Airgap
 	r.Path("/api/v1/app/airgap").Methods("OPTIONS", "POST", "PUT").HandlerFunc(handlers.UploadAirgapBundle)
 	r.Path("/api/v1/app/airgap/status").Methods("OPTIONS", "GET").HandlerFunc(handlers.GetAirgapInstallStatus)
+	r.Path("/api/v1/kots/airgap/reset/{appSlug}").Methods("OPTIONS", "POST").HandlerFunc(handlers.ResetAirgapInstallStatus)
```

Comment:
won't this be unreachable because of https://github.com/replicatedhq/kots/pull/1001/files#diff-79cb665f28a31d9d02f7e30a6aec9869R95

Reply:
yep, updated